### ISSUE TITLE
[SDL 0267] Main Menu UI Updates

### ISSUE THREAD

**Issue by theresalech** [smartdevicelink/sdl_java_suite#1246]
Proposal: [Main Menu UI Updates](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0267-addcommand-ui-updates.md)

> This proposal is to update the main menu `AddCommand` and `AddSubMenu` RPCs to allow for the main menu to have the same UI capabilities as `Choice` in `PerformInteraction` popup menus.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/882

Steering Committee Decision:

>The Steering Committee voted to accept this proposal with the following revision: additional text params should go to `MenuParams`.
> The proposal .md file was updated to reflect these revisions on 12/17/19.

**Comment by jordynmackool** [comment]
The proposal markdown file has been updated per the revisions included in the accepted [Revise SDL 0267 Main Menu UI Updates](https://github.com/smartdevicelink/sdl_evolution/issues/1114).

Accepted Revisions:
> Move parameters `secondaryText` and `tertiaryText` from the request body to the MenuParams struct.



> Update the proposed HMI API Changes to the following:

> ## HMI API Changes

> ###### Common.MenuParams

```xml
<struct name="MenuParams">
    <!-- New Parameters -->
    <param name="secondaryText" maxlength="500" type="String" mandatory="false">
        <description>Optional secondary text to display</description>
    </param>
    <param name="tertiaryText" maxlength="500" type="String" mandatory="false">
        <description>Optional tertiary text to display</description>
    </param>
</struct>
```

###### AddSubmenu
```xml
<function name="UI.AddSubMenu" messagetype="request">
    <!-- New Parameters -->
    <param name="secondaryImage" type="Image" mandatory="false">
        <description>Optional secondary image struct for menu cell</description>
    </param>
</function>
```

###### AddCommand
```xml
<function name="UI.AddCommand" messagetype="request">
    <!-- New Parameters -->
    <param name="secondaryImage" type="Image" mandatory="false">
        <description>Optional secondary image struct for menu cell</description>
    </param>
</function>
```

**Comment by RHenigan** [comment]
Closed with #1565
